requirements.txt

```diff
@@ -44,4 +44,4 @@ nbconvert==5.6.1
 pre-commit==2.13.0
 plotly==5.1.0
 dataframe-image==0.1.1
-
+h5netcdf==0.11.0
```

Comment:
Why did you add this one? Cause I don't see it used anywhere or am I missing something?

Reply:
@caldwellst why do we need this?